test/mocks/erc20/ERC20Mock.sol

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: GPL-3.0-or-later
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Why are we using `UNLICENSED` for mocks?

Reply:
It doesn't really matter what we're using here as long it's an open-source license. I reverted the changes to use GPL v3 again.

FWIW, my rationale was that nobody will install Sablier as a dependency just to get access to these mocks.